src/index.js

```diff
@@ -80,6 +81,7 @@ export default class Spinner extends React.Component {
 
   static defaultProps = {
     visible: false,
+    cancelable: true,
```

Comment:
@SandroMachado I agree, should be false and we should update Readme too if it isn't already